infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -105,7 +105,7 @@
                     + " -k <seqcache>, --sequencer-cache-size=<seqcache>                         "
                     + "               The size of the sequencer's cache. [default: 250000].\n    "
                     + " -B <size> --batch-size=<size>                                            "
-                    + "              The read/write batch size used for data transfer operations [default: 100].\n"
+                    + "              The read/write batch size used for data transfer operations [default: 20].\n"
```

Comment:
LR also has a CorfuServer, called `CorfuInterClusterReplicationServer`, which is mostly a copy of this one and also has default batch size of 100, probably need to take care of that one as well.

Reply:
fixed